SecretManagement.LastPass.Extension/SecretManagement.LastPass.Extension.psm1

```diff
@@ -20,13 +20,24 @@ function Invoke-lpass {
         $InputObject
     )
 
-    $lpassCommand = if ($null -ne $AdditionalParameters.lpassCommand){$AdditionalParameters.lpassCommand} else {'lpass'}
+    $lpassCommand = if ($null -ne $AdditionalParameters.lpassCommand){$AdditionalParameters.lpassCommand} else {''}
+    $lpassPath = if ($null -ne $AdditionalParameters.lpassPath){"`"$($AdditionalParameters.lpassPath)`""} else {'lpass'}
```

Comment:
Theses are the changes from #7 since without them, nothing can be tested on the Set-Secret end, they will disappear from the PR once #7 get merged (I will transpose any changes made in the other PR here as needed)